### ISSUE TITLE
React UI toolbars are not announced to screen readers

### ISSUE THREAD

**Issue by jbalsas** [liferay/alloy-editor#202]
In the YUI UI, whenever a toolbar was shown/hidden, it was announced to screen readers. This is now missing in the React UI
